draft-irtf-cfrg-randomness-improvements.md

```diff
@@ -91,36 +103,39 @@ normative:
 
 Randomness is a crucial ingredient for TLS and related security protocols.
 Weak or predictable "cryptographically-strong" pseudorandom number generators (CSPRNGs)
-can be abused or exploited for malicious purposes. The Dual EC random number
-backdoor and Debian bugs are relevant examples of this problem. This document describes a way for
-security protocol participants to mix their long-term private key into the entropy pool(s) from 
-which random values are derived. This augments and improves randomness from broken or otherwise
-subverted CSPRNGs.
+can be abused or exploited for malicious purposes. The Dual EC random number backdoor and Debian bugs 
+are relevant examples of this problem. 
+An initial entropy source that seeds a CSPRNG might be weak or broken as well, that can also lead to critical problems with the overall security. 
```

Comment:
"An initial entropy source that seeds a CSPRNG might be weak or broken as well, that can also lead to critical problems with the overall security."

to

"An initial entropy source that seeds a CSPRNG might be weak or broken as well, which can also lead to critical and systemic security problems."